examples/for-tests-react-16/src/AppWithReactDomRouter.js

```diff
@@ -1,14 +1,42 @@
 import React, { useState } from "react";
```

Comment:
need to update google one tap example app (if this does not work on chrome, try firefox)